cmd/baton-jira/main.go

```diff
@@ -38,9 +38,12 @@ func main() {
 func getConnector(ctx context.Context, v *viper.Viper) (types.ConnectorServer, error) {
 	l := ctxzap.Extract(ctx)
 
+	projectKeys := v.GetStringSlice("jira-project-keys")
+
 	builder := connector.JiraBasicAuthBuilder{
 		Base: &connector.JiraOptions{
-			Url: v.GetString("jira-url"),
+			Url:         v.GetString("jira-url"),
```

Comment:
```suggestion
			Url:         v.GetString(jiraUrlField.FieldName),
```

Ah, I see we do this throughout main.go. :/

Reply:
@ggreer thank you fixed all these in f79884da686e7395470cc0c7ddfb536092828745